src/main/java/uk/gov/hmcts/reform/ccd/client/model/SearchCriteria.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.ccd.client.model;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class SearchCriteria {
+
+    private final ObjectMapper mapper;
+
+    public SearchCriteria(ObjectMapper mapper) {
+        this.mapper = mapper;
+    }
+
+    public String matchAllQuery() {
+        JsonNode query = mapper.createObjectNode()
+            .put("size", 10)
+            .set("query", mapper.createObjectNode()
+                .set("match_all", mapper.createObjectNode()));
+
+        return query.toString();
+    }
+
+    public String searchByQuery(String key, String value) {
+        JsonNode query = mapper.createObjectNode()
+            .put("size", 10)
```

Comment:
what happens if there is more than 10 results?

Reply:
Is this `size` param to override default value of `NN` that CCD uses ? If that's the case then:

1. if `size` is not passed then use the default of CCD ( which means don't need to pass size at all in our query)
2. if we want to override it, can pass `size`